Mods/ArchipelagoPlayer.SC2Mod/enUS.SC2Data/LocalizedData/GameStrings.txt

```diff
@@ -2993,6 +2994,7 @@ Button/Tooltip/AP_zergmeleeweapons2=Further upgrades all Zerg melee attacks.
 Button/Tooltip/AP_zergmeleeweapons3=Maximizes all Zerg melee attacks.
 Button/Tooltip/AP_zergmeleeweapons4=Maximizes all Zerg melee attacks.
 Button/Tooltip/AP_zergmeleeweapons5=Maximizes all Zerg melee attacks.
+Button/Tooltip/AP_ZergMercenaryCompound=Trains Specialist units.

```

Comment:
Should we use "calls in" or "spawns" here? Mercs don't have a train time, so this feels a little off.

Reply:
I just copied from merc compound, I had the same question, but figured I'd stick with 'official' wording